service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ColleaguesToNotifyController.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import io.swagger.annotations.Api;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.enums.SolicitorRole;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Child;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.interfaces.WithSolicitor;
+import uk.gov.hmcts.reform.fpl.service.CaseRoleLookupService;
+import uk.gov.hmcts.reform.fpl.utils.CaseDetailsMap;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static uk.gov.hmcts.reform.fpl.utils.CaseDetailsMap.caseDetailsMap;
+
+@Api
+@RestController
+@RequestMapping("/callback/add-colleagues-to-notify")
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class ColleaguesToNotifyController extends CallbackController {
+
+    private final CaseRoleLookupService caseRoleLookupService;
+
+    @PostMapping("/about-to-start")
+    public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody CallbackRequest callbackrequest) {
+        final CaseDetails caseDetails = callbackrequest.getCaseDetails();
+        final CaseData caseData = getCaseData(caseDetails);
+
+        List<SolicitorRole> roles = caseRoleLookupService.getCaseSolicitorRolesForCurrentUser(caseData.getId());
+        List<WithSolicitor> represented = roles.stream()
+            .map(role -> role.getRepresentedPerson(caseData))
+            .filter(Optional::isPresent)
+            .map(Optional::get)
+            .map(Element::getValue)
+            .collect(Collectors.toList());
+
+        List<String> errors;
+        if (represented.isEmpty()) {
+            errors = List.of("There is no one this user is representing on this case.");
+            return respond(caseDetails, errors);
+        }
+
+        // Use the first party we are representing - solicitors shouldn't be representing multiple
+        // todo - what about children's solicitors...
```

Comment:
is the `todo` for the future ticket?

Reply:
oops left that comment in by accident - turns out childrens solicitors will be fine as even if we take the first child they represent we then copy the colleagues across all children to keep them upto date!